web/js/modules/geosearch/selectors.js

```diff
@@ -0,0 +1,200 @@
+import OlPoint from 'ol/geom/Point';
```

Comment:
A lot of the functions in this file don't strike me as true "selectors".  A selector is a function that accepts Redux state as an argument and returns data that is derived from that state.  The only function here that looks like it might fit this description would be `areCoordinatesWithinExtent`

[This article gives a great explanation of how and why to use selectors as well as how to memoize them with Reselect for better performance.](https://blog.isquaredsoftware.com/2017/12/idiomatic-redux-using-reselect-selectors/)